src/Corax/IndexWriter.cs

```diff
@@ -504,6 +508,19 @@ public bool TryDeleteEntry(string key, string term)
             return true;
         }
 
+        private void UnlikelyGrowBuffer(out Span<byte> buffer, out Span<Token> tokens, int newBufferSize, int newTokenSize)
+        {
+            Debug.Assert(buffer.Length <= newBufferSize);
```

Comment:
That is valid? It is an out param - it _has_ no length.

Reply:
Good catch